keyless/pepper/service/src/keyless_config.rs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::watcher::{unhexlify_api_bytes, ExternalResource};
+use anyhow::{anyhow, Result};
+use aptos_infallible::RwLock;
+use aptos_types::keyless::Configuration;
+use once_cell::sync::Lazy;
+use serde::{Deserialize, Serialize};
+use std::sync::Arc;
+
+#[derive(Serialize, Deserialize, Clone, Debug, Default, PartialEq)]
+pub struct TrainingWheelsPubKey {
+    vec: Vec<String>,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, Default, PartialEq)]
+pub struct OnChainKeylessConfiguration {
+    /// Some type info returned by node API.
+    pub r#type: String,
+    pub data: ConfigData,
+}
+
+impl OnChainKeylessConfiguration {
+    pub fn to_rust_repr(&self) -> Result<aptos_types::keyless::Configuration> {
+        let training_wheels_pubkey = self
+            .data
+            .training_wheels_pubkey
+            .vec
+            .first()
+            .map(|v| unhexlify_api_bytes(v.as_str()))
+            .transpose()
+            .map_err(|e| anyhow!("to_rust_repr() failed with unhexlify err: {e}"))?;
+        // We so far only consume `training_wheels_pubkey`.
```

Comment:
I'm a bit confused: why are you ignoring all the other fields?

The pepper service may not need them yet but why write incorrect code?

Reply:
fixed!